### PR TITLE
Clean up .gitignore from recipe

### PR DESCRIPTION
**Pull request by ioanok**
The .gitignore configuration from the recipe is outdated and has a lot of trash, a cleanup is required for accuracy.

### CODE REVIEW

File: shopware/core/6.4/manifest.json [hunk 1]
```diff
@@ -51,36 +51,13 @@
...
         "/files/*",
+        "!/files/.htaccess",
         "/var/*",
-        "!/var/log/.gitkeep",
-        "!/var/cache/.gitkeep",
+        "!/var/.htaccess",
         "/auth.json",
         "/install.lock",
-        "public/asset-manifest.json",
-        "files/asset-manifest.json"
```
Comment (shyim) [inline shopware/core/6.4/manifest.json:83]: I think this should stay, as some older Shopware versions sometimes write into files. (only 6.4 version)

File: shopware/core/6.4/manifest.json [hunk 2]
```diff
@@ -51,36 +51,13 @@
...
-        "/public/media/*",
-        "!/public/media/.gitkeep",
-        "/public/theme/*",
-        "!/public/theme/.gitkeep",
-        "/public/thumbnail/*",
-        "!/public/thumbnail/.gitkeep",
-        "/public/assets/*",
-        "/public/sitemap/*",
-        "!/public/sitemap/.gitkeep",
-        "/public/bundles/*",
```
Comment (shyim) [inline shopware/core/6.4/manifest.json:74]: Is still public/bundles and so on completely ignored? 🤔 I don't see it

File: shopware/core/6.4/manifest.json (hunk 1 above)
Comment (ioanok) [reply shopware/core/6.4/manifest.json]: Understood, I added back files/asset-manifest.json

File: shopware/core/6.4/manifest.json (hunk 2 above)
Comment (ioanok) [reply shopware/core/6.4/manifest.json:74]: You are right, they should be ignored, but there is no .gitkeep inside so I only added the public folders back:
```
/public/bundles/*
/public/media/*
/public/theme/*
/public/thumbnail/*
/public/sitemap/*

```

Comment (shyim) [reply shopware/core/6.4/manifest.json:74]: sounds good 👍

**Comment by shyim** [comment]
Hey,

can you do the code-quality recipe change in a separate PR thank you! :)

**Comment by ioanok** [comment]
Hey, I'm so sorry about this. I thought it was already merged.
I reverted the code-quality recipe update

**Pull request closed without merging**